### PR TITLE
Update ctx.js

### PR DESCRIPTION
**Pull request by AdamRoyn**
send media function

## Your Pull Request Title
Pull request description. Describe what you changed or added here.

## Type
- [x] New feature.
- [ ] Bug Fix.
- [ ] Other:...

## Another details
- does this have anything to do with any of the issues? (if yes, tag the issue)

### CODE REVIEW

File: src/classes/ctx.js [hunk 1]
```diff
@@ -74,6 +74,22 @@ module.exports = class Ctx {
     this._client.sendMessage(jid, content, options);
   }
 
+  async sendAudio(url, vn) {
+      this._client.sendMessage(this.id, { audio: { url: url }, mimetype: 'audio/mp4', ptt: vn || false }, { quoted: this._msg })
```
Comment (JastinXyz) [inline src/classes/ctx.js:78]: i think you can add new jid parameter and pass it into sendMessage (like ctx.sendMessage function)
 
 e.g:
```js
ctx.sendAudio(jid, url, vn)
// etc
```

and i think you should remove the quoted, or add new parameter to send it with quote or not

Comment (JastinXyz) [reply src/classes/ctx.js:78]: and with other functions too

Comment (AdamRoyn) [reply src/classes/ctx.js:78]: ok

**Comment by JastinXyz** [comment]
thanks for contributing!

**Pull request merged**